Ballot - remaining items (#117)
HL7/bulk-data: input/pagecontent/changes.md

* Added optional `includeAssociatedData` kickoff parameter and ValueSet for clients to indicate a set of pre-defined resources to omit or include with the response
* Provided guidance on server handling of unsupported kickoff parameters when a `prefer: handling=lenient header` is or is not provided
* Added recommended approach for clients to obtain historical data on new group members when not automatically included by server in Group level requests

> * Clarified that resources associated with groups containing non-patient members (such as practitioners) may be exported using a group-level bulk export request

Reviewer: ```suggestion
* Clarified that resources associated with groups containing non-patient members (e.g., groups of practitioners or groups of devices) may be exported using a group-level bulk export request
```

#### Export - Data
* Clarified that resource references in the response may be relative or absolute
* Provided guidance for servers and clients to send and retrieve `Binary` resources and `Attachment` elements

> * Changed requirement to populate Attachment.contentType in Attachments from a requirement to a recommendation to align with the core FHIR spec

Reviewer: ```suggestion
* Changed requirement to populate `Attachment.contentType` in Attachments from a requirement to a recommendation to align with the core FHIR spec
```

#### Export - Other
* Added recommendations on server capability documentation

#### Backend Services Authorization
* Migrated and integrated documentation into the SMART App Launch Implementation Guide
* Clarified that servers must support clients that provide a URL pointing to a JWKS Set on registration, as well as those that provide a JWKS Set directly on registration
* Clarified authorization requirements for status and data requests

> * Corrected errors in key verification algorithm documentation

Reviewer: ```suggestion
* Clarified the algorithm for verifying a client's public key
```
